pkg/handlers/session.go

```diff
@@ -75,6 +75,12 @@ func requireValidSession(kotsStore store.Store, w http.ResponseWriter, r *http.R
 	}
 
 	if time.Now().After(sess.ExpiresAt) {
+		if err := kotsStore.DeleteSession(sess.ID); err != nil {
+			err := errors.Wrapf(err, "session expired. failed to delete expired session %s", sess.ID)
+			response := ErrorResponse{Error: err.Error()}
+			JSON(w, http.StatusInternalServerError, response)
```

Comment:
should we just log here and continue?

Reply:
that is what I had. But later changed it to Internal Server Error after looking at the logout flow.
but log and continue(returns 401) would be a better option

Reply:
@emosbaugh changed to logging in https://github.com/replicatedhq/kots/pull/2706/commits/bbbb57fe68071689624b9a5a3851a3fa450b9706#diff-8be07e05f305ae1ebec9b3760f46190803066daf92afb41de47e5adf9e182837R80